src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -704,12 +705,16 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             if (_lo.get_optimization_attributes().use_onednn_impls) {
                 if (input.is_type<reshape>() || input.is_type<concatenation>())
                     return;
-
+                if (activation_func == cldnn::activation_func::relu_negative_slope &&
```

Comment:
AFAIK, `relu_negative_slope` includes both leaky relu(where slope is a scalar) and prelu(where slope is a vector). Does onednn not support both cases?

Reply:
No, my expectation is only for Prelu with vector, I''ll update it.

Reply:
Applied